bot_manager.py

```diff
@@ -92,13 +103,23 @@ def run(self):
 
             self.interlocked_exchange_fn(ctypes.byref(player_input_lock), ctypes.c_long(REFRESH_NOT_IN_PROGRESS))
 
+            if self.save_data and game_tick_packet.gameInfo.bRoundActive and old_time is not 0 and not old_time == current_time:
+                self.game_file.writelines(str(game_tick_packet) + '\n')
```

Comment:
Need good format for output writing since this is going to be a standard.  Some good discussion in Discord on this.

Reply:
Talked about writing cstruct bytes to file and reading back to struct with ctypes pack and unpack in the discord as a possibility: https://docs.python.org/3/library/struct.html#struct.unpack

Reply:
Note that I mistakenly linked you to struct whereas I am using a ctypes struct.  You can go back and forth from a bytearray to a ctypes struct though.